keen/explorer
# Use latest Keen-Dataviz fropm npm v1.0.3

## What does this PR do?
It upgrades to the latest version of Keen-Dataviz.js.

## How can I test this?
Run the test suite and do some QA to ensure that visualization works as expected.
